07-workbase/00-overview.md

```diff
@@ -35,10 +35,12 @@ then { (less: $x, more: $z) isa greater-than; };
   Note that this can have a performance impact, so in production we recommend only leaving the relation variables as required.
 </div>
 
-## Dependencies
+## Version Compatibility
 
 | Workbase       | Grakn Core          | Grakn Cluster          |
 | :------------: | :-----------------: | :-----------------: |
+| 2.0.2          | 2.0.2               | N/A                 |
+| 2.0.0, 2.0.1   | 2.0.0, 2.0.1        | N/A                 |
```

Comment:
I think Workbase actually does work with Cluster, we should check with Ganesh or Rui.